test/jdk/javax/net/ssl/SSLEngine/Basics.java

```diff
@@ -53,7 +57,21 @@ public class Basics {
             System.getProperty("test.src", "./") + "/" + pathToStores +
                 "/" + trustStoreFile;
 
+    private static final String TLS13_CIPHER_SUITE = "TLS_AES_256_GCM_SHA384";
+    private static final String TLS13_PROTOCOL = "TLSv1.3";
+
+    private static final String TLS12_PROTOCOL = "TLSv1.2";
```

Comment:
It looks like that the three constants are not used any longer?

Reply:
+1. These are to be passed as parameter to runTest. Either define and use all as constant or remove these.